frontend/src/styles/settings/themeOverride.scss

```diff
@@ -54,28 +54,28 @@
     text-decoration: none !important;
   }
   &:hover {
-    color: $default-light-green;
-    border: 1px solid $default-light-green;
+    color: $pagination-item-hover;
+    border: 1px solid $pagination-item-border-hover;
   }
 }
 
 .ant-btn-primary {
-  color: $white;
+  color: $pure-white;
   &:hover {
-    color: $white;
-    background-color: $default-light-green;
-    border: 1px solid $default-light-green;
+    color: $pure-white;
+    background-color: $primary-btn-color-hover;
+    border: 1px solid $primary-btn-border-color-hover;
   }
 }
 
 .ant-btn-dashed {
   margin: 10px 0;
-  color: $default-dark-green;
-  border-color: $default-dark-green;
+  color: $tertiary-btn-color;
+  border-color: $tertiary-btn-border-color;
   border-style: dashed;
   &:hover {
-    color: $default-light-green;
-    border-color: $default-light-green;
+    color: $tertiary-btn-color-hover;
```

Comment:
are the border-color and color the same? 
could `tertiary-btn-color` & `tertiary-btn-color-hover` work for both properties?

same for primary and secondary.

Reply:
disregard, I see the difference now.